ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/clientlibs/js/checkoutform.js

```diff
@@ -791,7 +791,6 @@ function setAddressDefaults() {
     getUserDeatils['street'] =streetArr;
     getUserDeatils['country_id'] = 'NA';
     getUserDeatils['region_code'] = 'NA';
-    getUserDeatils['email'] = 'NA';
```

Comment:
@anchal-a : are we not getting email as part of response ?

Reply:
Previously we were providing email as "NA" so the mail was not triggering for store pickup>guest user. After removing this, it is taking that email id which user is filling in email section at checkout page. So the email is triggering now.